python-3.12.yaml

```diff
@@ -93,7 +88,22 @@ test:
         python3.12 CVE-2023-27043-unittest.py
 
 subpackages:
-  - name: "python-3.12-doc"
+  - name: "python-3.12-default"
+    description: "python-3.12 as /usr/bin/python3"
+    dependencies:
+      provides:
+        - python3=${{package.full-version}}
+        - python-3=${{package.full-version}}
+      runtime:
+        - python-3.12
+    pipeline:
+      - runs: |
+          mkdir ${{targets.subpkgdir}}/usr/bin
```

Comment:
`-p` please

Reply:
i don't know. -p not necessary because subpkgdir starts out empty.... hm... but why did this work? i'll add -p

Reply:
Ci builds pipeline failed as /usr/ is not there and it's like no such directory.